SQLSolutions

```diff
@@ -0,0 +1,118 @@
+//Question 1
+  a. SELECT title FROM movies;
+  b. SELECT director FROM movies;
+  c. SELECT title, year FROM movies;
+  d. SELECT * FROM movies;
+
+//Question 2
+  a.  SELECT * FROM movies
+      WHERE id = 6
+
+  b. SELECT * FROM movies
+WHERE year BETWEEN 2000 AND 2010
+
+  c. SELECT * FROM movies
+WHERE year  NOT BETWEEN 2000 AND 2010
+
+  d. SELECT * FROM movies
+WHERE Year BETWEEN 1995 AND 2003
+
+//Question 3:
+  a. SELECT * FROM movies
+WHERE Title LIKE 'Toy Story%'
+
+  b. SELECT * FROM movies
+WHERE Director LIKE 'John%'
+
+  c. SELECT * FROM movies
+WHERE Director NOT LIKE 'John%'
+
+  d. SELECT * FROM movies
+WHERE Title LIKE 'WALL-%'
+
+//Question 4
+  a.SELECT distinct director
+FROM movies
+order by director ASC
```

Comment:
`SELECT DISTINCT director
FROM movies
ORDER BY director ASC`
in order to stay consistent you need to capitalize all the words that are part of SQL